src/Build/NuGet.Build.Packaging.Tasks/NuGet.Build.Packaging.Tasks.csproj

```diff
@@ -26,6 +26,11 @@
     <Content Include="MSBuild\Microsoft.Common.Targets\ImportAfter\Microsoft.NuGet.Packaging.ImportAfter.targets">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="NuGet.Build.Packaging.CrossTargetting.targets">
+      <SubType>Designer</SubType>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <PackagePath>buildCrossTargetting\%(Filename)%(Extension)</PackagePath>
```

Comment:
This is where the [.NET SDK is placing the cross-targeting files](https://github.com/dotnet/sdk/commit/215e323fad3b9c4a9d08301959002e5b247deb02).